### PR TITLE
Fork the sophus repositories.

### PR DESCRIPTION
**Pull request by clalancette**
We've been waiting for almost a year to get some fixes to sophus merged so that we can get turtlebot2 released into Humble and Iron: https://github.com/stonier/sophus/pull/23

For now, use a fork under my username to do the release. I still haven't merged that fix in, but once we switch this over I will have the power to do so.

While we are in here, switch the source and doc branches to be correct (we are releasing from the 1.3.x branch nowadays).

### CODE REVIEW

**Review by audrow — APPROVED** [review]

**Comment by audrow** [comment]
Looks good to me. @clalancette, I'll let you merge it in, just in case you want to wait on any other reviews.

**Comment by clalancette** [comment]
@stonier FYI.

**Comment by clalancette** [comment]
I'm going to go ahead with this for now so we can start getting some packages released and built.

**Pull request merged**
